### DISCUSSION TITLE
Add "Inform or Send to" options when adding an annotations

### DISCUSSION THREAD

**Discussion by henjoe** [grafana/grafana#53705] [Dashboards, Panels & Visualizations]
It would be great that when adding annotations in Grafana Panel, we have the options to send or inform someone through an email notifications that the annotation was added, and link to that panel is also added in the content of the email (similar to email alerts).

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
